.travis.yml

```diff
@@ -35,8 +35,16 @@ matrix:
    - node_js: "6"
      env: KARMA=true REACT=16
   allow_failures:
-    - node_js: "6"
-      env: KARMA=true REACT=16
+   - node_js: "6"
```

Comment:
these all need another space of indentation

Reply:
lol WAT. how did that happen